packages/create-expo/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### 🐛 Bug fixes
 
+- - Mark compressed `.gz` files as binary to avoid corruption when unpacking with `create-expo --template` ([#26741](https://github.com/expo/expo/pull/26741) by [@shirakaba](https://github.com/shirakaba))
```

Comment:
```suggestion
- Mark compressed `.gz` files as binary to avoid corruption when unpacking with `create-expo --template` ([#26741](https://github.com/expo/expo/pull/26741) by [@shirakaba](https://github.com/shirakaba))
```